docs/USE_CASES.md

```diff
@@ -93,7 +112,8 @@ SUBJECT_ID  XX    1    SAMPLEIDR    9    /samples/relapse9_1.fastq.gz    /sample
 
 ## Starting from realignement
```

Comment:
See if this paragraph and the one below should be removed?

Reply:
Same